<launch>
  <node pkg="test_node"
    type="test_decomp"
    name="test_decomp"
    output="screen">
    <param name="bag_file" value="$(find test_node)/data/iss_obs.bag"/>
    <param name="bag_topic" value="/obs_points"/>
    <param name="path_file" value="$(find test_node)/data/path.txt"/>
    <!--<param name="topic" value="/mob/mapper/zones"/>-->
  </node>

</launch>
